scripts/build-packages.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+const fs = require("fs");
+const shell = require("shelljs");
+
+shell.cd(path.join(__dirname, ".."));
+
+const packages = fs
+    .readdirSync(path.join(__dirname, "..", "packages"))
+    .filter(name => !name.startsWith("."));
```

Comment:
```suggestion
    .filter(name => name.startsWith("flat-"));
```

Reply:
Packages under `packages` should be considered valid packages conventionally. Prefixing `"flat-"` could lead to hard-to-debug bug in the long run.